applications/user-api/src/main/resources/logback.xml

```diff
@@ -0,0 +1,47 @@
+<configuration>
```

Comment:
Hva er dette konfigurasjon for ? Kan ikke huske å ha sett logback.xml i prosjektet før.

Reply:
Det er standard introdusert i api-cat. 
Den konfigurasjon tilbyr to log formater - json og compact. Vi må bruke json format i google cloud (satt som default), men i docker-compose og intellij, vi liker best compact format. Man må sette env variabel i docker-compose og intellij til dette.

Vi skulle faktisk introdusere det en gang og i alle tjenestene. Det er litt viktig at vi kunne se log til alle tjenestene i google cloud. Teknisk gjeld.

Reply:
https://github.com/Informasjonsforvaltning/fdk/issues/1371